lib/rules/web/admin_console/4.3/node.xyaml

```diff
@@ -85,3 +85,8 @@ add_tolerations:
   action: set_affinity_operator
   action: input_node_affinity_modal_value
   action: set_affinity_effect
+click_to_machines_page:
```

Comment:
The URL for machines page are `k8s/ns/openshift-machine-api/machine.openshift.io~v1beta1~Machine`, the  namespace name is hard coded now, I'm afraid it may change in future so I visit Machines page by clicking